openpype/hosts/blender/hooks/pre_add_run_python_script_arg.py

```diff
@@ -0,0 +1,62 @@
+from pathlib import Path
+
+from openpype.lib import PreLaunchHook
+from openpype.settings.lib import get_project_settings
+
+
+class AddPythonScriptToLaunchArgs(PreLaunchHook):
+    """Add python script to be executed before Blender launch."""
+
+    # Append after file argument
+    order = 15
+    app_groups = [
+        "blender",
+    ]
+
+    def execute(self):
+        # Check enabled in settings
+        project_name = self.data["project_name"]
+        project_settings = get_project_settings(project_name)
+        host_name = self.application.host_name
+        host_settings = project_settings.get(host_name)
+        if not host_settings:
+            self.log.info(f"""Host "{host_name}" doesn\'t have settings""")
+            return None
```

Comment:
```suggestion
            return
```
Does this really need the explicit `None` there?